### PR TITLE
Final full scenario

### PR DESCRIPTION
**Pull request by kmatlaha**
Final full scenario

### CODE REVIEW

File: tests/buyProducts_test.js [hunk 1]
```diff
@@ -46,6 +46,12 @@ Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPag
         console.log('VAT price is: ' + vatPrice);
         let totalPrice = await checkoutPage.getTotalPrice();
         console.log('Total price is: ' + totalPrice);
+        let response = await I.sendGetRequest('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?valcode=USD&json');
+        I.seeResponseCodeIs(200);
+        console.log('Response is: ' + response.data[0].rate);
+        let usdRate = response.data[0].rate;
+        let priceInUah = totalPrice * usdRate;
+        console.log("Price in UAH is: " + +priceInUah.toFixed(2));
```
Comment (adudyak) [inline tests/buyProducts_test.js]: move API request and price conversion to helper.

**Review by adudyak — CHANGES_REQUESTED** [review]

File: tests/buyProducts_test.js [hunk 2]
```diff
@@ -46,6 +47,9 @@ Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPag
         console.log('VAT price is: ' + vatPrice);
         let totalPrice = await checkoutPage.getTotalPrice();
         console.log('Total price is: ' + totalPrice);
+        let usdRate = await I.convertCurrencyTo('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?valcode=USD&json');
```
Comment (adudyak) [inline tests/buyProducts_test.js]: There should be no api links in scenario, move it to helper. Create one method `convertUsdToUah()` and use it.

**Review by adudyak — CHANGES_REQUESTED** [review]

File: tests/buyProducts_test.js (hunk 2 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: Fixed.

File: tests/buyProducts_test.js (hunk 1 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: Was done in previous commit and checked.

File: helpers/convertCurrency_helper.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const Helper = require('@codeceptjs/helper');
+
+class ConvertCurrency extends Helper {
+  async convertUsdToUah() {
```
Comment (adudyak) [inline helpers/convertCurrency_helper.js]: According to method's name, it should convert, but it only returns current rate. Add input parameter - price in USD and change method to do conversion.

**Review by adudyak — CHANGES_REQUESTED** [review]

File: helpers/convertCurrency_helper.js (hunk 3 above)
Comment (kmatlaha) [reply helpers/convertCurrency_helper.js]: Changed.

File: tests/buyProducts_test.js [hunk 4]
```diff
@@ -44,11 +45,13 @@ Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPag
         console.log('Eco tax price is: ' + ecoTaxPrice);
         let vatPrice = await checkoutPage.getVatPrice();
         console.log('VAT price is: ' + vatPrice);
-        let totalPrice = await checkoutPage.getTotalPrice();
-        console.log('Total price is: ' + totalPrice);
+        let totalPriceInUsd = await checkoutPage.getTotalPrice();
+        console.log('Total price in USD is: ' + totalPriceInUsd);
+        let totalPriceInUah = await I.convertUsdToUah(totalPriceInUsd);
+        console.log("Total price in UAH is: " + totalPriceInUah.toFixed(2));
```
Comment (adudyak) [inline tests/buyProducts_test.js]: move `.toFixed(2)` to `I.convertUsdToUah()`

**Review by adudyak — APPROVED** [review]

**Pull request merged**